.NET/Microsoft.Recognizers.Text.DateTime/Hindi/Extractors/HindiDateTimeAltExtractorConfiguration.cs

```diff
@@ -0,0 +1,68 @@
+﻿using System.Collections.Generic;
+using System.Text.RegularExpressions;
+using Microsoft.Recognizers.Definitions.Hindi;
+
+namespace Microsoft.Recognizers.Text.DateTime.Hindi
+{
+    public class HindiDateTimeAltExtractorConfiguration : BaseDateTimeOptionsConfiguration, IDateTimeAltExtractorConfiguration
```

Comment:
No need to implement/test cases for this specific type until all basic DateTime items are complete.

Reply:
> 
> 
> > Skips in the DateParser test are related to "yesterday|tomorrow" which in Hindi are the same and are disambiguated only by context. कल and परसों are ambiguous as they refer to one day later and two days later respectively both forward and backward in time; this has to be disambiguated based on the context and other words indicating grammatical tense/aspect of the sentence.
> 
> Could any frequent usage cases of these be disambiguated by common terms immediately next to them?

There are such terms, we will add a couple of cases with such.